.github/workflows/osx.yml

```diff
@@ -69,49 +69,49 @@ jobs:
           mkdir -p kokkos/{build,install}
           cd kokkos/build
           cmake \
-          -DKokkos_ENABLE_${{ matrix.backend }}=ON \
-          -DCMAKE_CXX_FLAGS="-Werror" \
-          -DCMAKE_CXX_STANDARD=17 \
-          -DKokkos_ENABLE_COMPILER_WARNINGS=ON \
-          -DKokkos_ENABLE_DEBUG_BOUNDS_CHECK:BOOL=${{ matrix.debug_bounds_check }} \
-          -DKokkos_ENABLE_DEPRECATED_CODE_3=OFF \
-          -DKokkos_ENABLE_TESTS=OFF \
-          -DKokkos_ENABLE_DEPRECATED_CODE_4=OFF \
-          -DCMAKE_BUILD_TYPE=${{ matrix.cmake_build_type }} \
-          -DCMAKE_INSTALL_PREFIX=$PWD/../install \
-          ..
+          -S "$PWD/.." \
+          -B "$PWD" \
+          -D CMAKE_CXX_FLAGS="-Werror" \
+          -D CMAKE_CXX_STANDARD=17 \
```

Comment:
Do we know if the compiler in this workflow supports c++20? We might want to consider changing this soon if so

Reply:
I'll add it in the PR and if does not pass we will know hahaha